README.md

```diff
@@ -1,29 +1,39 @@
 # Nix RFCs (Request For Comments)
 
-Many changes, including bug fixes and documentation improvements can be
+Many ideas, including bug fixes and documentation improvements can be
 implemented and reviewed via the normal GitHub pull request workflow.
 
-Some changes though are "substantial", and we ask that these be put through a
-bit of a design process and produce a consensus among the Nix community.
+Some ideas though are "substantial", and we ask that these be put through a
```

Comment:
Not sure why we're replacing changes with ideas throughout this document. This seems like a potentially contentious choice, with little upside, so I'd rather leave this out.

Reply:
As stated above, probably the main purpose of _informational_ RFCs is to record consensus on something that does not primarily originate in a change.

My best word for that something has been: "idea". Maybe there is a better word?